package by.it.orlov.jd02_06;

import java.io.IOException;

public class Runner {

    public static void main(String[] args) throws IOException {

        Log.putLog("проверка ошибки 1");
        Log.putLog("проверка ошибки 2");
        Log.putLog("и еще одна проерка 3");
    }
}
